File: app/src/main/java/com/karanchuk/roman/testtranslate/data/database/model/Meaning.java
package com.karanchuk.roman.testtranslate.data.database.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by roman on 18.6.17.
 */

public class Meaning {
    @SerializedName("text")
    private String mText;

    public String getText() {
        return mText;
    }

    public void setText(String text) {
        mText = text;
    }

    @Override
    public String toString() {
        return mText;
    }
}

File: app/src/main/java/com/romankaranchuk/translator/data/database/model/Meaning.kt
package com.romankaranchuk.translator.data.database.model

import com.google.gson.annotations.SerializedName

/**
 * Created by roman on 18.6.17.
 */
class Meaning {
    @SerializedName("text")
    var text: String? = null
    override fun toString(): String {
        return text!!
    }
}